google/cloud/storage/client.py

```diff
@@ -928,7 +928,7 @@ def generate_signed_post_policy_v4(
         credentials=None,
         virtual_hosted_style=False,
         bucket_bound_hostname=None,
-        scheme=None,
+        scheme="http",
```

Comment:
It came to my attention that everywhere we're setting `scheme` to `"http"` by default, except POST policies. Doesn't seem to be correct.